src/components/CioPlpGrid/CioPlpGrid.tsx

```diff
@@ -85,6 +98,7 @@ export default function CioPlpGrid(props: CioPlpGridWithRenderProps) {
               {response?.results?.length ? (
                 <div className='cio-plp-grid'>
                   <div className='cio-filters-container cio-large-screen-only'>
+                    <Groups groups={response.groups} {...groupsConfigs} />
```

Comment:
Should it be named `GroupsFilters` to avoid collision when we add actual `Groups` later for browse

Also, what do you think of just making it just Filters? and Filters render all type of filters including group filters.

Reply:
I did think about it but we discussed this and I'm pretty sure we decided to have them as separate pieces 😓